src/app/components/cookie-policy/cookie-policy.component.html

```diff
@@ -16,150 +17,154 @@ <h1 class="govuk-heading-xl" id="cookies">Cookies</h1>
-            </tr>
-          </tbody>
-        </table>
-        <h3 class="govuk-heading-m" id="5-to-make-the-service-more-secure">5. To make the service more secure</h3>
-        <p>We set cookies which prevent attackers from modifying the contents of the other cookies we set. This makes the service more secure and protects your personal information.</p>
-        <table class="govuk-table">
-          <thead class="govuk-table__head">
+            </thead>
+            <tbody>
+            <tr class="govuk-table__row" *ngFor="let cookie of cookiesByCat(IDENTIFY)">
+              <td class="govuk-table__cell">{{cookie.name}}</td>
+              <td class="govuk-table__cell">{{cookie.purpose}}</td>
+              <td class="govuk-table__cell">{{cookie.expires}}</td>
+            </tr>
+            </tbody>
+          </table>
+        </ng-container>
+        <ng-container *ngIf="SECURITY">
+          <h3 class="govuk-heading-m" id="5-to-make-the-service-more-secure">To make the service more secure</h3>
+          <p>We set cookies which prevent attackers from modifying the contents of our pages and which prevent
+            unauthorised access to the service. This
+            makes the service more secure and protects your personal information.</p>
+          <table class="govuk-table">
+            <thead class="govuk-table__head">
             <tr class="govuk-table__row">
               <th class="govuk-table__header">Name</th>
               <th class="govuk-table__header">Purpose</th>
               <th class="govuk-table__header">Expires</th>
             </tr>
-          </thead>
-          <tbody>
-            <tr class="govuk-table__row">
-              <td class="govuk-table__cell">TSxxxxxxxx</td>
-              <td class="govuk-table__cell">Protects your session from tampering</td>
-              <td class="govuk-table__cell">When you close your browser</td>
-            </tr>
-            <tr class="govuk-table__row">
-              <td class="govuk-table__cell">__state</td>
-              <td class="govuk-table__cell">Identifies you to the service and secures your authentication</td>
-              <td class="govuk-table__cell">When you close your browser</td>
-            </tr>
-            <tr class="govuk-table__row">
-              <td class="govuk-table__cell">X_CMC</td>
-              <td class="govuk-table__cell">Helps us keep track of your session</td>
-              <td class="govuk-table__cell">When you close your browser</td>
-            </tr>
-            <tr class="govuk-table__row">
-              <td class="govuk-table__cell">_csarf</td>
-              <td class="govuk-table__cell">Used in probate applications and the money claims service to help protect
-                against forgery</td>
-              <td class="govuk-table__cell">When you close your browser</td>
+            </thead>
+            <tbody>
+            <tr class="govuk-table__row" *ngFor="let cookie of cookiesByCat(SECURITY)">
```

Comment:
And sixth repetition of this bit too... apart from the `SECURITY` parameter. Can we get a table component for these cookie listings?
```
<table class="govuk-table">
  <thead class="govuk-table__head">
    <tr class="govuk-table__row">
      <th class="govuk-table__header">Name</th>
      <th class="govuk-table__header">Purpose</th>
      <th class="govuk-table__header">Expires</th>
    </tr>
  </thead>
  <tbody>
    <tr class="govuk-table__row" *ngFor="let cookie of cookies">
      <td class="govuk-table__cell">{{cookie.name}}</td>
      <td class="govuk-table__cell">{{cookie.purpose}}</td>
      <td class="govuk-table__cell">{{cookie.expires}}</td>
    </tr>
  </tbody>
</table>
```
Where `cookies` is an `@Input()` on the component:
`<cookie-policy-listing [cookies]="cookiesByCat(SECURITY)"></cookie-policy-listing>`

Reply:
I really didn't want to do any more than was necessary. Yes there is some repetition in the HTML file, but it's not worth the effort of changing it